src/Assignment6/Bankingsystem/EnumOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// <copyright file="EnumOptions.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Bankingsystem
+{
+    /// <summary>
+    /// Has the enum for type ofaccounts.
+    /// </summary>
+    public class EnumOptions
+    {
+        /// <summary>
+        /// Account type enum has the types of account.
+        /// </summary>
+        public enum AccountType
+        {
+            /// <summary>
+            /// Savings account has restrictions in withdrawal.
+            /// </summary>
+            Savingsaccount = 1,
```

Comment:
Camel case here also.

Reply:
The CSharpStarter Template shows error when the elements of Enum begins with lower case. So, it is changed to Pascal Case.